lib/ConnectionManager.js

```diff
@@ -614,8 +614,8 @@ ConnectionManager.prototype.onPacketQueueReadable = function () {
                 header.type !== jute.OP_CODES.PING &&
                 header.type !== jute.OP_CODES.AUTH) {
 
+            this.xid = (this.xid % 2147483647) + 1;
```

Comment:
I think we just need to set the initial value of XID to 1.

because
a: The role of xid is to ensure the order of requests, so we should not break its meaning.
b:  `2 ** 32 -1` I believe this value can meet most needs

Reply:
With a continuously running application doing on average 100 requests per second (mine, which is not a huge one, already does way more), that would break after 248.5 days.